public/app/features/alerting/unified/utils/rule-form.ts

```diff
@@ -365,6 +366,57 @@ export function rulerRuleToFormValues(ruleWithLocation: RuleWithLocation): RuleF
   }
 }
 
+export function grafanaRuleDtoToFormValues(rule: RulerGrafanaRuleDTO, namespace: string): RuleFormValues {
+  const defaultFormValues = getDefaultFormValues();
+
+  const ga = rule.grafana_alert;
+  const duration = rule.for;
+  const annotations = rule.annotations;
+  const labels = rule.labels;
+  if (rulerRuleType.grafana.recordingRule(rule)) {
```

Comment:
Could we extract the common properties between both types into an object? And then spread that onto each, along with `defaultFormValues`, so we don't have too much duplication 
(i.e. name, annotations, labels etc are mostly the same)